File: sdk/typescript/src/governance.ts
export async function login(email: string, baseUrl?: string) {
  const url = `${baseUrl || process.env.NEXT_PUBLIC_GOVERNANCE_URL || ''}/auth/login`;
  const res = await fetch(url, {
    method: 'POST',
    headers: { 'Content-Type': 'application/json' },
    body: JSON.stringify({ email })
  });
  const data = await res.json();
  if (!res.ok) throw new Error(data.detail || 'Failed to login');
  return data;
}

export async function listUsers(baseUrl?: string) {
  const url = `${baseUrl || process.env.NEXT_PUBLIC_GOVERNANCE_URL || ''}/rbac/users`;
  const res = await fetch(url);
  const data = await res.json();
  if (!res.ok) throw new Error(data.detail || 'Failed to list users');
  return data.users;
}

export async function listAuditEvents(baseUrl?: string) {
  const url = `${baseUrl || process.env.NEXT_PUBLIC_GOVERNANCE_URL || ''}/audit/events`;
  const res = await fetch(url);
  const data = await res.json();
  if (!res.ok) throw new Error(data.detail || 'Failed to list events');
  return data.events;
}

File: sdk/python/apex_sdk/governance.py
"""Python SDK for the governance service."""

from __future__ import annotations

from typing import Any, Dict, List
from .client import APEXClient


class GovernanceAPI:
    def __init__(self, client: APEXClient) -> None:
        self._client = client

    async def login(self, email: str) -> Dict[str, Any]:
        res = await self._client.post(self._client.governance_url, "/auth/login", {"email": email})
        res.raise_for_status()
        return res.json()

    async def list_users(self) -> List[Dict[str, Any]]:
        res = await self._client.get(self._client.governance_url, "/rbac/users")
        res.raise_for_status()
        return res.json().get("users", [])

    async def list_audit_events(self) -> List[Dict[str, Any]]:
        res = await self._client.get(self._client.governance_url, "/audit/events")
        res.raise_for_status()
        return res.json().get("events", [])